docs/adr/0014-standardized-provenance-recording.md

```diff
@@ -0,0 +1,62 @@
+<!--
+SPDX-FileCopyrightText: 2025 German Aerospace Center (DLR), Forschungszentrum Jülich, Helmholtz-Zentrum Dresden-Rossendorf
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+# Standardized provenance recording
+
+* Status: proposed
+* Deciders: sdruskat, skernchen, notactuallyfinn
+* Date: 2025-10-17
+
+Technical story: 
+* https://github.com/softwarepub/hermes/pull/442
+* https://github.com/softwarepub/hermes/issues/363
+
+## Context and Problem Statement
+
+To consolidate traceability of the metadata, and resolution based on metadata sources in case of duplicates, etc., we need to record the provenance of metadata values in a __standardized__ way.
+Additionally we use the [PROV-O ontology](https://www.w3.org/TR/prov-o/) and [JSON-LD](https://www.w3.org/TR/json-ld/) and want that HERMES records as much of the provenance as possible to not overcomplicate plugin development.
+
+To do this, we need to specify what provenance information is recorded and how it can be implemented in HERMES to make it easy to use.
+
+## Considered Options
+
+* Provide HERMES API-methods that also document themselves
+
+## Decision Outcome
+
+Chosen option: "Provide HERMES API-methods that also document themselves", because comes out best.
```

Comment:
```suggestion
```

As this is in the proposal stage, I think we need more feedback on the actual solution. What are alternatives for this solution? This ADR is already a very good track record of our thinking so far, but I think we need more buy-in before making the actual decision.

That said, this isn't a blocker for the PR, it just says: we need a decision about this further down the line.